File: scripts/inspect-revenue-xlsx.js
const XLSX = require('xlsx');
const fs = require('fs');
const path = require('path');

const base = 'C:\\Users\\user\\Desktop\\브라더 공유폴더\\★BREM 경리일보★';
const files = fs.readdirSync(base).filter(f => f.endsWith('.xlsx') && f.includes('★'));

for (const file of files) {
  const full = path.join(base, file);
  console.log('\n========', file, '========');
  const wb = XLSX.readFile(full, { cellDates: true });
  console.log('Sheets:', wb.SheetNames.join(' | '));
  for (const name of wb.SheetNames) {
    const rows = XLSX.utils.sheet_to_json(wb.Sheets[name], { header: 1, defval: '' });
    console.log('\n---', name, `(rows: ${rows.length}) ---`);
    rows.slice(0, 30).forEach((row, i) => {
      const cells = row.slice(0, 15).map(v => String(v ?? '').replace(/\s+/g, ' ').trim());
      if (cells.some(Boolean)) console.log(String(i + 1).padStart(3), cells.join(' | '));
    });
  }
}

File: scripts/inspect_revenue_xlsx.py
# -*- coding: utf-8 -*-
import os
from openpyxl import load_workbook

base = r"C:\Users\user\Desktop\브라더 공유폴더\★BREM 경리일보★"
for file in sorted(os.listdir(base)):
    if not file.endswith(".xlsx") or "★" not in file:
        continue
    path = os.path.join(base, file)
    print("\n========", file, "========")
    wb = load_workbook(path, data_only=True, read_only=True)
    print("Sheets:", " | ".join(wb.sheetnames))
    for name in wb.sheetnames:
        ws = wb[name]
        print(f"\n--- {name} ---")
        count = 0
        for i, row in enumerate(ws.iter_rows(max_row=35, max_col=15, values_only=True), 1):
            cells = ["" if v is None else str(v).strip() for v in row]
            if any(cells):
                print(f"{i:3}", " | ".join(cells))
                count += 1
        if count == 0:
            print("(empty)")
    wb.close()
